spyder/plugins/editor/widgets/editor.py

```diff
@@ -1396,6 +1396,19 @@ def set_run_cell_copy(self, state):
         """If `state` is ``True``, code cells will be copied to the console."""
         self.run_cell_copy = state
 
+    def set_current_project_path(self, root_path=None):
+        """
+        Set the current active project root path.
+
+        Parameters
+        ----------
+        root_path: str or None, optional
+            Path to current project root path. Default is None.
+        """
+        for index in range(self.get_stack_count()):
+            editor = self.tabs.widget(index)
+            editor.set_current_project_path(root_path)

```

Comment:
Why not using here `for finfo in self.data:`? That's what we use to iterate among CodeEditors in almost all methods in `EditorStack`.

Reply:
Fixed